src/librustc_metadata/rmeta/mod.rs

```diff
@@ -277,6 +277,7 @@ define_tables! {
     super_predicates: Table<DefIndex, Lazy!(ty::GenericPredicates<'tcx>)>,
     mir: Table<DefIndex, Lazy!(mir::Body<'tcx>)>,
     promoted_mir: Table<DefIndex, Lazy!(IndexVec<mir::Promoted, mir::Body<'tcx>>)>,
+    unused_generic_params: Table<DefIndex, Lazy<FiniteBitSet<u64>>>,
```

Comment:
Extraneous space in `FiniteBitSet<u64 >`.

Reply:
Uh, I don't see that extraneous space.

Reply:
For posterity: 
![image](https://user-images.githubusercontent.com/77424/87793181-966d3d80-c84d-11ea-9bc2-619cd5e51260.png)

Resolving this as I think it's a clientside rendering bug on my side.